samples/apps/copilot-chat-app/WebApp/README.md

```diff
@@ -1,34 +1,41 @@
+
+## Requirements to run this app
+
+1. Azure Open AI Service Key and working End point. (https://learn.microsoft.com/en-us/azure/cognitive-services/openai/quickstart?tabs=command-line&pivots=programming-language-studio)
+2. A registered App in Azure Portal (https://learn.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+   - Select Single-page application (SPA) as platform type, and the Redirect URI will be http://localhost:3000
+   - Select **`Accounts in any organizational directory (Any Azure AD directory - Multitenant) and personal Microsoft accounts (e.g. Skype, Xbox)`** as the supported account type for this sample.
+   - Note the **`Application (client) ID`** from your app registration.
+3. Yarn - used for installing the app's dependencies (https://yarnpkg.com/getting-started/install)
+
 ## Running the sample
 
-1. Ensure the SKWebApi running at `https://localhost:40443` (or whichever host and port you chose)
-3. You will also need to
-   [register your application](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app)
-   in the Azure Portal. Follow the steps to register your app
-   [here](https://learn.microsoft.com/azure/active-directory/develop/quickstart-register-app).
-    - Select **`Single-page application (SPA)`** as platform type, and the Redirect URI will be **`http://localhost:3000`**
-    - Select **`Accounts in any organizational directory and personal Microsoft accounts`** as supported account types for this sample.
-4. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where
-   `APP_CHAT_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from the Azure Portal and
-   `APP_BACKEND_URI=` is the URI where your backend is running:
-        
-        APP_CHAT_CLIENT_ID=
-        APP_BACKEND_URI=https://localhost:40443
-
-5. **Run** the following command `yarn install` (if you have never run the app before)
-   and/or `yarn start` from the command line.
-6. A browser will automatically open, otherwise you can navigate to `http://localhost:3000/` to use the ChatBot.
-
-## About the Copilot Chat Sample
-
-### Authentication in this sample
-This sample uses the Microsoft Authentication Library (MSAL) for React to sign in users. Learn more about it here: https://learn.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-react.
\ No newline at end of file
+1. Ensure the SKWebApi is running at `https://localhost:40443`. See [SKWebApi README](../SKWebApi/README.md) for instructions.
+2. Create an **[.env](.env)** file to this folder root with the following variables and fill in with your information, where
+   `REACT_APP_CHAT_CLIENT_ID=` is the GUID copied from the **Application (client) ID** from your app registration in the Azure Portal and
+   `REACT_APP_BACKEND_URI=` is the URI where your backend is running.
+
+   REACT_APP_BACKEND_URI=https://localhost:40443
```

Comment:
Please add a trailing /

The way we concatenate the backend host:port with the path chokes if there is none.

We should probably look into that (I suspect it's currently just a string concatenation) but at least having the app working with the example data would be a start